plugins/libplugin-pay.c

```diff
@@ -188,6 +219,43 @@ static struct command_result *payment_createonion_success(struct command *cmd,
 	return command_still_pending(cmd);
 }
 
+static struct command_result *payment_createonion_success(struct command *cmd,
+							  const char *buffer,
+							  const jsmntok_t *toks,
+							  struct payment *p)
+{
+	struct out_req *req;
+	struct route_hop *first = &p->route[0];
+	struct secret *secrets;
+	p->createonion_response = tal_createonion_response_from_json(p, buffer, toks);
```

Comment:
.. and handle NULL response here?  crash is fine, as long as we get that msg out first!

Reply:
Calling `plugin_err` so we exit with a message.